lib/Workflow/Persister/DBI.pm

```diff
@@ -244,19 +244,15 @@ sub create_workflow {
 sub fetch_workflow {
     my ( $self, $wf_id ) = @_;
     $self->_init_fields();
-    my $sql = q{
-        SELECT %s, %s
-          FROM %s
-         WHERE %s = ?
-    };
+    my $sql = q{SELECT %s, %s FROM %s WHERE %s = ?s};
```

Comment:
@ehuelsmann is there a `s` too many in the end of the quoted SQL string?

Reply:
Addressed.